docs/reference/concepts/code-generation.md

```diff
@@ -1,69 +1,19 @@
-[//]: # (Doc of docs reference 31)
-[//]: # (TODO - Tech writer review)
-[//]: # (TODO - Move this doc from ECS to generic - and change its title to not be ECS)
-[//]: # (TODO - use discussions about content in here https://docs.google.com/document/d/1IGblyE-pvA4ZyJIjN8PcD1Ct6pE4FNhtlXRdp_Sy97o/edit)
-<%(TOC)%>
-# Code generation
- _This document relates to both the [MonoBehaviour and ECS workflows]({{urlRoot}}/reference/workflows/which-workflow)._
+<%(TOC max="2")%>
+# Code generator
 
-> Note this document refers to ECS components but it is also relevant if you are using the MonoBehaviour workflow. You don't need to know about ECS to use component generation.
+The code generator uses `.schema` files to generate C# code that you can use to interact with SpatialOS in Unity. See the [schemalang reference](https://docs.improbable.io/reference/latest/shared/schema/introduction#schema-introduction) for details on how to create schema components.
 
-The code generator uses `.schema` files to generate components that the Unity ECS can understand. See the [schemalang guide](https://docs.improbable.io/reference/latest/shared/schema/introduction#schema-introduction) for details on how to create schema components.
+The code generator runs when you:
```

Comment:
might just be me, but this reads as though it runs only when you do all of the following

Reply:
Can make it more explicit: `... runs when you do one of the following:`